includes/course-theme/class-sensei-course-theme-option.php

```diff
@@ -122,6 +122,8 @@ public function should_use_sensei_theme() {
 			return false;
 		}
 
+		$course_id = absint( $course_id );
```

Comment:
The ideal would make `get_current_course` return an `int` instead of string because the documentation says that it should return `int`. But I decided to not change that to avoid other issues.